armi/bookkeeping/db/databaseInterface.py

```diff
@@ -390,7 +390,10 @@ def getHistory(
         if nowRequested:
             for param in params or history.keys():
                 if param == "location":
-                    history[param][now] = tuple(comp.spatialLocator.indices)
+                    # might save as int or np.int64, so forcing int keeps things predictable
+                    history[param][now] = tuple(
+                        int(i) for i in comp.spatialLocator.indices
+                    )
```

Comment:
Same as the above comment on np.int64

Sometimes the `.indices` returned are int64 and those get processed as such into a string during `writeAssemHistory`. (so you end up qith `np.int64(0)` as a string rather than just `0`). 

Anyway, this is where I found it made a difference to force an int. Again, not a root cause solution.